internal/pkg/mapper/configmetadata/remote.go

```diff
@@ -19,6 +19,28 @@ func (m *configMetadataMapper) OnRemoteChange(changes *model.RemoteChanges) erro
 		m.onRemoteLoad(objectState, metadataMap)
 	}
 
+	// Process saved objects
+	if len(changes.Saved()) > 0 {
+		api, err := m.StorageApi()
+		if err != nil {
+			return err
+		}
+		pool := api.NewPool()
+		for _, objectState := range changes.Saved() {
+			config, ok := objectState.RemoteState().(*model.Config)
+			if !ok {
+				continue
+			}
+			if len(config.Metadata) > 0 {
+				pool.Request(api.UpdateConfigMetadataRequest(config)).Send()
```

Comment:
Tady ale není `ChangedFields`, tak posílám update vždycky když jsou nastavený nějaký metadata. Je to ok?

Reply:
Dobra poznamka, v tomto PR to sprav tak. 
A pozriem ako tam dostat changedFields (nemal by to byt problem), a upravim to v samostatnom PR.